Sources/Fuzzilli/Core/ProgramBuilder.swift

```diff
@@ -665,9 +665,9 @@ public class ProgramBuilder {
     }
 
     /// Append a splice from another program.
-    public func splice(from program: Program, at index: Int) {
+    public func splice(from program: Program, at index: Int, activeContext: Context) {
```

Comment:
Initially I'd left this in there so that it's easier to specify arbitrary context when writing tests for this function. But now that I think about it we can get rid of it since tests for non-default contexts requires mocking up a host program and then calling splice within the right context. Similar to how it's being done for `testClassSplicing`. So yeah this param is redundant